<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Clone</class>
    <widget class="QWidget" name="Clone">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>441</width>
                <height>162</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Clone</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="IconLabel" name="cloneWidget" native="true">
                    <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                            <horstretch>0</horstretch>
                            <verstretch>0</verstretch>
                        </sizepolicy>
                    </property>
                </widget>
            </item>
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                    <item>
                        <widget class="QTabWidget" name="tabWidget">
                            <property name="currentIndex">
                                <number>0</number>
                            </property>
                            <widget class="QWidget" name="https">
                                <attribute name="title">
                                    <string>HTTPS</string>
                                </attribute>
                                <layout class="QHBoxLayout" name="horizontalLayout_3">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QLineEdit" name="httpsUrlLineEdit"/>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="httpsCopyButton">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                            <widget class="QWidget" name="ssh">
                                <attribute name="title">
                                    <string>SSH</string>
                                </attribute>
                                <layout class="QHBoxLayout" name="horizontalLayout_2">
                                    <property name="spacing">
                                        <number>0</number>
                                    </property>
                                    <item>
                                        <widget class="QLineEdit" name="sshUrlLineEdit">
                                            <property name="readOnly">
                                                <bool>true</bool>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="sshCopyButton">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string/>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                            <widget class="QWidget" name="cloneTab">
                                <attribute name="title">
                                    <string>CLONE</string>
                                </attribute>
                                <layout class="QVBoxLayout" name="verticalLayout_2">
                                    <item>
                                        <widget class="QLabel" name="savePathLabel">
                                            <property name="font">
                                                <font>
                                                    <bold>true</bold>
                                                </font>
                                            </property>
                                            <property name="text">
                                                <string>Save path:</string>
                                            </property>
                                        </widget>
                                    </item>
                                    <item>
                                        <layout class="QHBoxLayout" name="horizontalLayout_4">
                                            <item>
                                                <widget class="QLineEdit" name="repoSavePathLineEdit">
                                                    <property name="toolTip">
                                                        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Path where
                                                            the repository will be saved.&lt;/p&gt;&lt;p&gt;Ex:
                                                            C:\Users\User&lt;/p&gt;&lt;p&gt;Cloning automatically
                                                            creates a folder with the name of the repository.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
                                                        </string>
                                                    </property>
                                                </widget>
                                            </item>
                                            <item>
                                                <widget class="QPushButton" name="selectDirectoryButton">
                                                    <property name="text">
                                                        <string>Select directory</string>
                                                    </property>
                                                </widget>
                                            </item>
                                        </layout>
                                    </item>
                                    <item>
                                        <widget class="QPushButton" name="cloneButton">
                                            <property name="sizePolicy">
                                                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                                                    <horstretch>0</horstretch>
                                                    <verstretch>0</verstretch>
                                                </sizepolicy>
                                            </property>
                                            <property name="text">
                                                <string>Clone</string>
                                            </property>
                                        </widget>
                                    </item>
                                </layout>
                            </widget>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>IconLabel</class>
            <extends>QWidget</extends>
            <header>widgets.IconLabel</header>
            <container>1</container>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
